components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -1136,6 +1146,13 @@ private void setSpOIDCProperties(Map<String, List<String>> spOIDCProperties, OAu
         String renewRefreshToken = getFirstPropertyValue(spOIDCProperties, RENEW_REFRESH_TOKEN);
         oauthApp.setRenewRefreshTokenEnabled(renewRefreshToken);
 
+        String authorizationCodeValidityPeriodProperty = getFirstPropertyValue(spOIDCProperties,
+                AUTHORIZATION_CODE_VALIDITY_PERIOD);
+        if (authorizationCodeValidityPeriodProperty != null) {
```

Comment:
If the property is not present, it would be better to return the server default value.